shared/utils/src/common.js

```diff
@@ -300,6 +301,24 @@ mclass[mclasscnvloss] = {
 	key: mclasscnvloss
 }
 
+export const mclasscnvAmp = 'CNV_amplification'
```

Comment:
please make a note this is next level above `mclasscnvgain` and is used in 5-category CNV data